ContestUtil/src/org/icpc/tools/contest/util/ImagesGenerator.java

```diff
@@ -59,7 +59,10 @@ public class ImagesGenerator {
 	private static final Rectangle TEXT = new Rectangle(370, 795, 1275, 230);
 
 	private static final Dimension DESKTOP = new Dimension(1920, 1080);
-	private static final Dimension ICON = new Dimension(56, 56);
+	private static final Dimension[] ICONS = new Dimension[]{
```

Comment:
Why make icons an array instead of just adding a "DOMJUDGE_ICON" (naming tbd)?

Reply:
Because I couldn't come up with a clean name besides exactly DOMJUDGE_ICON. If you are fine with that, I can do it.

Reply:
Feel free to using something like BIGGER_ICON if you prefer, but at least that name doesn't need a comment to remember why it's there. 😆

FWIW, I had this change ready 30s after we chatted about it, but didn't PR b/c I thought you said you didn't want the non-padded ones and hadn't had time to investigate.

Reply:
Yeah we don't need the non-padded ones but they don't hurt I guess. We could change the `pad` boolean to an enum (later) so we can skip them.